tests/endtoend_tests/common.py

```diff
@@ -87,6 +87,7 @@ def build_tesseract(
     assert result.exit_code == 0, result.exception
 
     image_tags = json.loads(result.stdout.strip())
-    assert image_name in image_tags
```

Comment:
Why remove this?

Reply:
It was replaced by `tag_exists(image_name, image_tags)` that parses it correctly: `foo/bar in repo/foo/bar` should be true, but `foo/bar in blafoo/bar` should not.